test/routes/noiseVibrationDocuments.route.test.js

```diff
@@ -49,7 +49,7 @@ lab.experiment('Noise and vibration emissions documents upload tests:', () => {
     const options = {
       pageHeading: 'Upload noise and vibration emissions documents',
       submitButton: 'Continue',
-      fileTypes: ['PDF', 'DOC', 'DOCX', 'ODT']
+      fileTypes: ['PDF', 'DOC', 'DOCX', 'ODT', 'ABS', 'BIN', 'CNA', 'DAT', 'DBF', 'DGM', 'FIL', 'FMS', 'FMT', 'GEO', 'GOT', 'GRF', 'SHP', 'SHX']
```

Comment:
We seem to be adding more complexity rather than making it easier by having a long, complicated list of allowed file types.
We will look at taking a different approach for pages where we are asking for a potentially wide range of technical files (ticket to be raised).